Sources/SharedModels/User/Account.swift

```diff
@@ -6,6 +6,7 @@ public struct Account: Codable {
     public let login: String
     public let name: String
     public let firstName: String
+    public let lastName: String?
```

Comment:
Is it optional? Otherwise maybe make it non-optional to stay consistent with the other fields here

Reply:
It can be `nil` as a part of the `user` field contained in `ProgrammingExerciseStudentParticipation`